.cproject

```diff
@@ -206,6 +207,7 @@
 						</toolChain>
 					</folderInfo>
 					<sourceEntries>
+						<entry excluding="ex2_hal/ex2_hyperion_solar_panel_software|ex2_hal/ex2_gps_software/ceedling-deprecated|ex2_hal/ex2_gps_software/test|ex2_hal/ex2_uhf_software/equipment_handler|libcsp/src/arch/macosx|libcsp/src/drivers/can/can_socketcan.c|ex2_services/ex2_demo_software|libcsp/src/drivers/usart/usart_linux.c|libcsp/src/drivers/usart/usart_windows.c|libcsp/examples|ex2_hal/ex2_sband_software/equipment_handler|libcsp/src/rtable/csp_rtable_cidr.c|libcsp/src/arch/posix|libcsp/src/arch/windows|libcsp/src/bindings" flags="VALUE_WORKSPACE_PATH|RESOLVED" kind="sourcePath" name=""/>

```

Comment:
It's weird that this line is added instead of CCS modifying line 212 in the same file. Anyway, ex2_hal/ex2_uhf_software/equipment_handler should not be excluded. See [here](https://github.com/AlbertaSat/ex2_obc_software/commit/18c8a8d527b98515a7f3b6cadcf3049deac8badd#diff-1ebe600e005d9c03f4e213f2ee0ca4c31b4e9df54c4ca3c7a8a352ef6b153ec2R208). Either you forgot to pull the changes from upstream or git is not doing its job.